frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json

```diff
@@ -252,7 +252,7 @@
   "The last saved values will be updated": "The last saved values will be updated",
   "The Storage Class will be used to request storage from the underlying infrastructure to create the backing persistent volumes that will be used to provide the OpenShift Container Storage (OCS) service.": "The Storage Class will be used to request storage from the underlying infrastructure to create the backing persistent volumes that will be used to provide the OpenShift Container Storage (OCS) service.",
   "The backing storage requested will be higher as it will factor in the requested capacity replica factor and fault tolerant costs associated with the requested capacity.": "The backing storage requested will be higher as it will factor in the requested capacity replica factor and fault tolerant costs associated with the requested capacity.",
-  "If you wish to use the Arbiter stretch cluster, a minimum of 4 nodes(2 different zones, 2 nodes per zone) and 1 additional zone with 1 node is required. All nodes must be pre-labeled with zones in order to be validated on cluster creation.": "If you wish to use the Arbiter stretch cluster, a minimum of 4 nodes(2 different zones, 2 nodes per zone) and 1 additional zone with 1 node is required. All nodes must be pre-labeled with zones in order to be validated on cluster creation.",
+  "If you wish to use the Arbiter stretch cluster, a minimum of 4 nodes (2 different zones, 2 zones per node) and 1 additional zone with 1 node is required. All nodes must be pre-labeled with zones in order to be validated on cluster creation.": "If you wish to use the Arbiter stretch cluster, a minimum of 4 nodes (2 different zones, 2 zones per node) and 1 additional zone with 1 node is required. All nodes must be pre-labeled with zones in order to be validated on cluster creation.",
```

Comment:
I'm doubted why "2 nodes per zone" are updated to "2 zones per node", in my understanding, a node only can be in one zone and a zone can contain many nodes.

Reply:
Thanks @yanpzhan. Tried to fix the spacing and messed it up. I'll fix it. :)